server/cri/v1/rpc_list_pod_sandbox_stats.go

```diff
@@ -0,0 +1,39 @@
+package v1
+
+import (
+	"context"
+
+	"github.com/cri-o/cri-o/server/cri/types"
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func (s *service) ListPodSandboxStats(
+	ctx context.Context, req *pb.ListPodSandboxStatsRequest,
+) (*pb.ListPodSandboxStatsResponse, error) {
+	r := &types.ListPodSandboxStatsRequest{}
```

Comment:
Seems this should belong to the next commit (the third one in the series, named "stub {List}PodSandboxStats" or so).

Reply:
or maybe not... I'm not quite following the logic